docs/tre-admins/identities/api.md

```diff
@@ -0,0 +1,73 @@
+# The API Identity
+
+## Name
+The API Identity is typically called `<TRE_ID> API` within the AAD Portal.
+
+## Purpose
+This identity's credentials are stored in the `core` Key Vault and mandatory for the running of the Trusted Research Environment (TRE). It is required for the API Application, hosted in Azure App Service, to authenticate to Azure Active Directory and authorize the various operations.
+
+## Application Roles
+
+| Display name | Description | Allowed member types | Value |
+| ------------ | ----------- | -------------------- | ----- |
+| TRE Administrators | Provides resource administrator access to the TRE. | Users/Groups,Applications | `TREAdmin` |
+| TRE Users | Provides access to the TRE application. | Users/Groups,Applications | `TREUser` |
+
+## Microsoft Graph Permissions
+| Name | Type* | Admin consent required |  TRE usage |
+| --- | -- | -----| --------- |
+| Directory.Read.All | Application | Yes | Allows the app to read directory objects (roles/permissions) in your organization's directory, such as roles and permissions, without a signed-in user. |
+| User.Read.All | Application | Yes | Allows the app to read user profiles without a signed in user to check that the user has permissions to execute an action e.g., to view workspaces. See `/api_app/services/aad_authentication.py`. |
+|email|Delegated|No|Used to read the user's email address when creating TRE resources|
+|openid|Delegated|No|Allows users to sign in to the app with their work or school accounts and allows the app to see basic user profile information.|
+|profile|Delegated|No|Used to read the user's profile when creating TRE resources|
+
+
+'*' See the difference between [delegated and application permission](https://docs.microsoft.com/graph/auth/auth-concepts#delegated-and-application-permissions) types. See [Microsoft Graph permissions reference](https://docs.microsoft.com/graph/permissions-reference) for more details.
+
+## Clients
+This identity should only be used by the API Application.
+
+## How to create
```

Comment:
These instructions are repeated across other .md files.
Why repeat them?

Reply:
There is a page for each identity that we create.

Reply:
Following chat I understand the reason for this repetition, and that this is a stepping stone to a place where the repetition is removed.